Makefile

```diff
@@ -6,6 +6,7 @@ OTEL_VERSION=main
 OTEL_STABLE_VERSION=main
 
 VERSION=$(shell git describe --always --match "v[0-9]*" HEAD)
+COREVERSION=$(shell git describe --always --match "v[0-9]*" HEAD | grep -o 'v[^-]*' | cut -c 2-)
```

Comment:
`CORE` in the context of this repository usually means https://github.com/open-telemetry/opentelemetry-collector

Maybe use something like TRIMMED_VERSION? I'm not sure what would make the most sense here.

Reply:
Also, can we reuse `VERSION` for the first half of this variable? This would make it a bit clearer and easier to maintain going forward.

Reply:
So i reused the `$(VERSION)` and tried to avoid using echo for this to get the code more clear.

sorry for causing trouble here :see_no_evil: